src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -758,15 +761,20 @@ protected function getRawObject(string $pdfData, int $offset = 0): array
                     $offset += 6;
                     if (1 == preg_match('/^([\r]?[\n])/isU', substr($pdfData, $offset, 4), $matches)) {
                         $offset += \strlen($matches[0]);
+
+                        $streamLen = (int) $this->getHeaderValue($headerDic, 'Length', 'numeric', 0);
+                        $skip = !$this->config->getRetainImageContent() && 'XObject' == $this->getHeaderValue($headerDic, 'Type', '/') && 'Image' == $this->getHeaderValue($headerDic, 'Subtype', '/');
```

Comment:
Replace the first part (`!$this->...`) with something like `false === `. It is more readable and will also improve maintenance in the future.

Reply:
done